lib/components/LinearProgressBar/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import clsx from 'clsx';
+import { Text } from '../Text';
+import { TGenericSizes } from "../../types";
+import "./LinearProgressBar.scss"
+
+type TLinearProgressProps = {
+    className?: string;
+    danger_limit:number;
+    is_loading: boolean;
+    label:React.ReactNode;
+    percentage:number;
+    size?: Extract<TGenericSizes, "lg" | "md" | "sm" | "xs">;
+    warning_limit:number;
+};
+
+export const LinearProgressBar = ({
+    className,
+    danger_limit,
+    is_loading,
+    label,
+    percentage,
+    size="xs",
```

Comment:
```suggestion
    labelSize="xs",
```
to be more clear where we are applying it